<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="yuanridandeMacBook-Pro.local" name="test2" tests="4" failures="2" timestamp="7 Jun 2017 03:34:54 GMT" time="0.017" errors="0">
  <testcase name="t1" time="0.002" classname="testng.TestListenerExample"/>
  <testcase name="t2" time="0.001" classname="testng.TestListenerExample">
    <failure type="org.testng.TestException" message="
Method TestListenerExample.t2()[pri:0, instance:testng.TestListenerExample@5c7fa833] should have thrown an exception of type class java.lang.RuntimeException">
      <![CDATA[org.testng.TestException: 
Method TestListenerExample.t2()[pri:0, instance:testng.TestListenerExample@5c7fa833] should have thrown an exception of type class java.lang.RuntimeException
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- t2 -->
  <testcase name="t3" time="0.0" classname="testng.TestListenerExample">
    <skipped/>
  </testcase> <!-- t3 -->
  <testcase name="t4" time="0.001" classname="testng.TestListenerExample">
    <failure type="java.lang.RuntimeException" message="Some other message">
      <![CDATA[java.lang.RuntimeException: Some other message
	at testng.TestListenerExample.t4(TestListenerExample.java:43)
Caused by: java.lang.RuntimeException: Some message
	at testng.TestListenerExample.t4(TestListenerExample.java:42)
	... 24 more
Caused by: java.lang.RuntimeException: Some message
	at testng.TestListenerExample.t4(TestListenerExample.java:41)
	... 24 more
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- t4 -->
</testsuite> <!-- test2 -->
